### PR TITLE
Add path option to createChild function

### PR DESCRIPTION
**Pull request by Nilbertocs**
This just gives the option to select the directory for the child domain

### CODE REVIEW

**Comment by usmannasir** [comment]
What if the path is not given?

**Comment by Nilbertocs** [comment]
> What if the path is not given?

It is being checked on line 923
If not given, the default is assumed.

```
if args.path:
    path = args.path
else:
    path = "public_html/" + args.childDomain
```

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ sh <(curl https://cyberpanel.net/install.sh || wget -O - https://cyberpanel.net/
 
 
 ```
-sh <(curl https://raw.githubusercontent.com/usmannasir/cyberpanel/stable/preUpgrade.sh || wget -O - https://raw.githubusercontent.com/usmannasir/cyberpanel/stable/preUpgrade.sh)
+sh <(curl https://raw.githubusercontent.com/NNetworkBR/cyberpanel/stable/preUpgrade.sh || wget -O - https://raw.githubusercontent.com/NNetworkBR/cyberpanel/stable/preUpgrade.sh)
```
Comment (usmannasir) [inline README.md]: Need to change all these refrences to official repo before i can go through this.

Comment (Nilbertocs) [reply README.md]: > Need to change all these refrences to official repo before i can go through this.

Reverted commits!

**Comment by usmannasir** [comment]
I've merged it to v2.3.4-dev, but I think i will have to fix something in it

**Comment by usmannasir** [comment]
OK. I think its arlight, good to go

**Pull request merged**